src/pre_defined_classes/simple_programs_taking_numeric_user_input/hard/q11/MainTest.java

```diff
@@ -0,0 +1,54 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.hard.q11;
+
+import global.BaseTest;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+
```

Comment:
Add comment to indicate what kind of question this is (Should be `// Parsons with distractors`)